tools/hangar/src/utils.ts

```diff
@@ -42,3 +48,14 @@ export function sanitize_json_paths(path: string) {
 
   return finalObj;
 }
+
+export function tfResourcesOf(templateStr: string): string[] {
+  return Object.keys(JSON.parse(templateStr).resource).sort();
+}
+
+export function tfResourcesOfCount(
```

Comment:
For other test utils you're using the actual cdktf lib, any reason these couldn't be included